piano-hero/mobile/src/main/java/com/novoda/pianohero/GameMvp.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.pianohero;
+
+interface GameMvp {
+
+    interface Model {
+
+        RoundViewModel startGame();
+
+        RoundViewModel onNotesPlayed(Callback callback, Note... note);
+
+        interface Callback {
+            void onSequenceComplete();
+        }
+    }
+
+    interface View {
+
+        void showRound(RoundViewModel viewModel);
+
+        void showGameComplete();
+
+        void showError();
```

Comment:
:x: we don't pass the error anymore - we lose the ability to show the user what the error was